suite-native/atoms/src/Button/Button.tsx

```diff
@@ -35,7 +40,7 @@ export type ButtonProps = Omit<PressableProps, 'style' | 'onPressIn' | 'onPressO
     size?: ButtonSize;
     style?: NativeStyleObject;
     isDisabled?: boolean;
-} & MergeExclusive<{ iconLeft?: IconName }, { iconRight?: IconName }> &
+} & MergeExclusive<{ viewLeft?: ButtonAccessoryType }, { viewRight?: ButtonAccessoryType }> &
```

Comment:
Why did you change the name fro icon to view? the icon seems to be more explanatory to me. We still want to send there only icons (even though they can be animated now). 

But maybe this is just my personal preference.

Reply:
I changed the name because we can put anything there now, not only icon